hooks/pre-commit

```diff
@@ -0,0 +1 @@
+./build
```

Comment:
If you decide to change the name of the build script from _build_ to _build.sh_, don't forget to change this.